### ISSUE TITLE
Switching patterns in Pattern mode

### ISSUE THREAD

**Issue by Adam-Vandal** [polyend/TrackerIssues#360]
in Patern mode, when I switch pattern 48 to 49, on the lighting touchpads are not asociated paterns 49 and more. There should be next page of patterns (and then next and next).
When I'm in the pattern 49 or more, and I press one of the lighting touchpads, it jumps to some of previous patterns (01-48), even they  are not highlited. More over it should IMHO jump to one of patterns between 49-96.

Please make there next pages of patterns on the lighting touchpads. 

Thank you.
![bug02_02](https://user-images.githubusercontent.com/74454887/99152899-18a72a80-26a5-11eb-919f-5b49a684c5db.jpg)
![bug02_01](https://user-images.githubusercontent.com/74454887/99152901-19d85780-26a5-11eb-8d80-003b27de8bf7.jpg)

**Comment by colonel-blimp** [comment]
Yes, please!  The PT pads' visual feedback and instant value input is so useful most of the time, but then when you go past pattern 48 they all go dark goes dark and pressing them does nothing. :/
